Tutorials_UsingAPIs/Flow-Control-Busy-Wait-Button/main.cpp

```diff
@@ -9,7 +9,7 @@ int main() {
     while(1) {
         if(BUTTON_PRESS == button){
            led = !led;
-           wait(1);
+           wait_us(1000000);
```

Comment:
Why is that a busy wait? The example is called busy-wait because we don't use semaphore etc, but we shouldn't push users towards busy loops.

Reply:
I convert it to the `wait_us()` because the whole example page is talking about busy wait.  if we change it to sleep, I think we need to chnage som contents to docs page as well:
https://os.mbed.com/docs/mbed-os/v5.14/tutorials/application-flow-control.html

Reply:
I agree with Bartek that we should limit usage of `wait_us` so we will have to rewrite the application flow control tutorial. I will add a task to work on this next sprint.

For this PR, I propose that we use `wait_us` only in the first example (Tutorials_UsingAPIs/Flow-Control-Busy-Wait/main.cpp ) and that we emphasize that busy-wait is to be avoided. All other examples should use `ThisThread::sleep_for`.

Here we should use `ThisThread::sleep_for(1000)` and rename the example Flow-Control-Active-Polling-Button.